projects/vdk-core/src/vdk/api/plugin/connection_hook_spec.py

```diff
@@ -5,6 +5,10 @@
 
```

Comment:
you have force-pushed new changes, and yet commits on this PR seem still 1?
let me review the latest changes,